bower.json

```diff
@@ -6,7 +6,7 @@
     "Ofir Dagan <ofirdagan@gmail.com>"
   ],
   "dependencies": {},
-  "main": "scripts/xdLocalStorage.js",
+  "main": "dist/scripts/xdLocalStorage.min.js",
```

Comment:
Hi @braxtondiggs , I'm sorry for the big delay from some reason the github mail reached my spam folder.. Now it's fixed.

Regarding your change, according to bower (http://bower.io/docs/creating-packages/) you should not reference minified files in the main section. Why do u need it?